packages/logging/package.js

```diff
@@ -17,11 +18,12 @@ Package.onUse(function (api) {
   // here due to this package's dependency on
   // `String.prototype.padRight` which is polyfilled only in
   // `ecmascript-runtime-client@0.6.2` or newer.
-  api.use(['ejson', 'ecmascript', 'ecmascript-runtime-client']);
+  api.use(['ejson', 'ecmascript', 'typescript', 'ecmascript-runtime-client']);
```

Comment:
@StorytellerCZ  This forces every app to depend on the `typescript` package, even when not using it, or when using an alternative.